File: benchmarks/fib30/fib30.zig
pub fn fib(n: u64) u64 {
    if (n <= 1) {
        return n;
    }
    return fib(n - 1) + fib(n - 2);
}

pub fn main() void {
    const n: u64 = 30;
    const result = fib(n);
    std.debug.print("{}\n", .{result});
}

File: benchmarks/fib30/fib30.kt
fun fib(n: Int): Long {
    if (n <= 1) {
        return n.toLong()
    }
    return fib(n - 1) + fib(n - 2)
}

fun main() {
    val n = 30
    val result = fib(n)
    println(result)
}
